.gitlab-ci.yml

```diff
@@ -57,7 +36,7 @@ deploy:
   dependencies:
     - build
   tags:
-    - docker
+    - deploy:advences
```

Comment:
@emiketic-admin @raed-emiketic This shouldn't be set to Advences

Reply:
I'll fix it now